### DISCUSSION TITLE
Versão 9.5.4

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#452] [Announcements]
Version 9.5.4:
- Changed donation button position.

Version 9.5.3:
- Fixed: The button to listen to the selected text overlays some of the text

Version 9.5.2:
- Fixed:
    - Link to translate via Yandex in new tab
    - Bing was not translating to these languages: _Hmong, Kurdish (Sorani), Mongolian, Norwegian and Serbian._
- Update interface translations

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.5.4'>Versão 9.5.4</a>.</em>

**Comment by verifyurhuman** [comment]
@FilipePS append the crx file to new release pls, it helps us get rid of all of those strange and annoying issues, otherwise I will have to use the old version insread.

**Reply by kingmanlove1** [reply]
The transparent color of the pop-up window is difficult to read, the best for me is dark black, as in the old version. Please add this option in future versions

**Reply by FilipePS** [reply]
> @FilipePS append the crx file to new release pls, it helps us get rid of all of those strange and annoying issues, otherwise I will have to use the old version insread.

Version 9.5.5 has just been released. CRX file added again.
https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.5.5

> The transparent color of the pop-up window is difficult to read, the best for me is dark black, as in the old version. Please add this option in future versions

In this version, the transparence of the selected text panel has been removed.

**Reply by verifyurhuman** [reply]
@FilipePS I'm glad you make it better and better!
